packages/ruleset/test/accept-parameter.test.js

```diff
@@ -7,7 +7,7 @@ const expectedSeverity = severityCodes.warning;
 const expectedErrorMsg =
   "Operations should not explicitly define the 'Accept' header parameter";
 
-describe('Spectral rule: accept-parameter', () => {
+describe(`Spectral rule: ${ruleId}`, () => {
```

Comment:
When working on this PR, I noticed that I had previously missed the testcase descriptions when modifying various rule id's.  Rather than change to another hard-coded value, we might as well just use the rule id that is configured in the testcase.
In this PR, I'll fix the testcases for the rules in this group, plus the testcases for the first group of rules.